task/summary/backfillrunner.go

```diff
@@ -181,5 +181,12 @@ func (t *BackfillTaskRunner) Run(ctx context.Context) error {
 	}
 	t.logger.Info(fmt.Sprintf("Backfilled %d BGM summaries", count))
 
+	t.logger.Debug("Starting User Continuous Summary Creation")
```

Comment:
This can be a loop over `[]string{"bgm", "cgm", "continuous"}`

Reply:
makes sense, done.